src/Jira/Api.php

```diff
@@ -156,13 +177,14 @@ public function setEndpoint($url)
 	 */
 	public function getFields()
 	{
-		if ( !count($this->fields) ) {
+		// Fetch fields when the method is called for the first time.
+		if ( $this->fields === null ) {
```

Comment:
No worries. Just use it consistently so we don't me different null checking styles in different places.

Yeah, the `empty` function is hell. I never use it.